tests/unit/states/archive_test.py

```diff
@@ -72,15 +67,9 @@ def test_extracted_tar(self):
                                                    'file.file_exists': mock_false,
                                                    'file.makedirs': mock_true,
                                                    'cmd.run_all': mock_run}):
-                    if HAS_PWD:
-                        running_as = pwd.getpwuid(os.getuid()).pw_name
```

Comment:
As you can see [here](https://github.com/terminalmage/salt/blob/49d8bd1dbec1fa5927a1e487e7f0799de2e2ee11/salt/states/archive.py#L130-L134), the username is not factored in here at all. In fact, when doing a ``git log -p salt/states/archive.py``, I can't find any point in the history of this state where this variable incorporated the username.